.gdbinit

```diff
@@ -14,6 +14,15 @@ break kernel_init
 break assert_fail
 break panic_fail
 
+define nextuser
```

Comment:
Could you provide a short description of this command that can be displayed with `help user`?

Reply:
Yup.